APMrover2/APMrover2.cpp

```diff
@@ -75,9 +75,19 @@ const AP_Scheduler::Task Rover::scheduler_tasks[] = {
     SCHED_TASK(accel_cal_update,       10,    100),
     SCHED_TASK(dataflash_periodic,     50,    300),
     SCHED_TASK(button_update,          5,     100),
+    SCHED_TASK(stats_update,           5,     100),
```

Comment:
Sure, what else is Rover going to do with its clocks?

Will change to 1 (like Copter)